source/bdDetect.py

```diff
@@ -413,6 +413,7 @@ def driverSupportsAutoDetection(driver):
 	"VID_0904&PID_600A", # Brailliant2 80
 	"VID_0904&PID_6201", # Vario 340
 	"VID_0483&PID_A1D3", # Orbit Reader 20
+	"VID_0904&PID_6301", # Vario 4

```

Comment:
This will make our linter happy

```suggestion
	"VID_0904&PID_6301",  # Vario 4
```